app/uk/gov/hmrc/servicedependencies/controller/ServiceDependenciesController.scala

```diff
@@ -56,6 +60,36 @@ class ServiceDependenciesController @Inject()(
       } yield Ok(Json.toJson(depsWithRules))
   }
 
+  def metaArtefactDependencies(
+                                   flag: SlugInfoFlag,
+                                   repoType: Option[RepoType],
+                                   group: Option[String],
+                                   artefact: Option[String],
+                                   versionRange: Option[BobbyVersionRange],
+                                   scope: Option[List[DependencyScope]]
+                                 ): Action[AnyContent] = Action.async { implicit request =>
+    for {
+      getTeams    <- teamsAndRepositoriesConnector.getTeamsForServices
+      getServices <- derivedDependencyRepository.findServicesByDeployment(flag, group, artefact, scope)
+      getOther    <- derivedDependencyRepository.findByOtherRepository(group, artefact, scope)
+    } yield {
+
+      val dependencies: Seq[MetaArtefactDependency] = repoType match {
+        case Some(Other)   => getOther
```

Comment:
The names `getOther`/`getServices` sound like retrievals, but they are the actual data.  We don't need to make both, so we could move it all into the for-comprehension
```scala
for {
  teams        <- teamsAndRepositoriesConnector.getTeamsForServices
  dependencies <- repoType match {
                     case Some(RepoType.ServiceType => derivedDependencyRepository.findServicesByDeployment(flag, group, artefact, scope)
                     case _     => derivedDependencyRepository.findByOtherRepository(group, artefact, scope)
                   }
  dependenciesByRange = ...
  dependenciesWithTeams = ...
    } yield ...
```
Note, it's actually just services that have special handling - all other types (primarily test and libraries) will go to the new repository